WordPress/Classes/ViewRelated/Site Creation/FinalAssembly/SiteAssemblyWizardContent.swift

```diff
@@ -61,29 +68,41 @@ final class SiteAssemblyWizardContent: UIViewController {
         navigationController?.isNavigationBarHidden = true
         setNeedsStatusBarAppearanceUpdate()
 
+        attemptSiteCreation()
+    }
+
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+
+        coordinator.animate(alongsideTransition: { [weak self] _ in
+            self?.contentView.adjustConstraints()
+        })
+    }
+
+    // MARK: Private behavior
+
+    private func attemptSiteCreation() {
         do {
             let wizardOutput = try siteCreator.build()
 
             contentView.domainName = wizardOutput.siteURLString
-            service.createSite(creatorOutput: wizardOutput) { [contentView] status in
-                contentView.status = status
+            service.createSite(creatorOutput: wizardOutput) { [weak self] status in
+                guard let strongSelf = self else {
+                    return
+                }
+
+                if .failed == status {
```

Comment:
This is just a personal preference, but I find these statements easier to read when they compare variable in the left hand side and value in the right hand side, like `if status == .failed`